scripts/ASTImportTest.sh

```diff
@@ -84,7 +84,12 @@ echo "Looking at $NSOURCES .sol files..."
 WORKINGDIR=$PWD
 
 # for solfile in $(find $DEV_DIR -name *.sol)
-for solfile in $(find "$SYNTAXTESTS_DIR" "$ASTJSONTESTS_DIR" -name *.sol)
+# boost_filesystem_bug specifically tests a local
+# fix for a boost::filesystem bug. Since the test
+# involves a malformed path, there is no point in
+# running AST tests on it.
+# See https://github.com/boostorg/filesystem/issues/176
```

Comment:
This is way below even the extremely conservative 80-char lines :)

```suggestion
# boost_filesystem_bug.sol specifically tests a local fix for a boost::filesystem bug. Since the test
# involves a malformed path, there is no point in running AST tests on it.
# See https://github.com/boostorg/filesystem/issues/176
```